tests/phpunit/integration/data/stories_in_amp.html

```diff
@@ -33,7 +33,7 @@
   <script async custom-element="amp-video" src="https://cdn.ampproject.org/v0/amp-video-0.1.js"></script>
   <title>Stories in AMP - Hello World</title>
   <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
-  <link rel="canonical" href="https://amp.dev/documentation/examples/introduction/stories_in_amp/index.html">
+  <link rel="canonical" href="https://wp.stories.google/stories/intro-to-web-stories-storytime/index.html">
```

Comment:
Technically this file here is an actual dump of the `https://amp.dev/documentation/examples/introduction/stories_in_amp/index.html` page. We don't need to change it.